filestore/filestore_test.go

```diff
@@ -3,6 +3,7 @@ package filestore
 import (
```

Comment:
I think this is another rebase that's in a weird place

Reply:
This file is still in master.  The changes are because the linter complained about not error checking